RCONPlugin/RCONPlugin.cpp

```diff
@@ -72,12 +72,10 @@ void RCONPlugin::on_message(server* s, websocketpp::connection_hdl hdl, message_
 			{
 				//is_allowed
 				//Rebuild command
-				std::stringstream new_command;
-				for (auto txt : inp)
-				{
-					new_command << "\"" << replace(txt, "\"", "\\\"") << "\" ";
-				}
-				gameWrapper->Execute([cmd = new_command.str(), &_cvarManager = cvarManager, log = *logRcon](GameWrapper* gw) {
+				std::string payload = msg->get_payload();
+				//Doesn't seem neccessary? words in "" come thru in payload and sendback ok
```

Comment:
I think this code exists in the case of a " being inside of quotes, like ```send_rcon(`mynotifier "my parameter has a \" in it!"`)``` should correctly get reconstructed. Have you tried the different behavior before and after making this change?

Reply:
I did mess around with it and being an edge case I'm not sure of what scenarios to look out for lol. I did test it a fair amount with python and I'll share those results here:
```
			await websocket.send('sendback "te"st"')
			time.sleep(0.1)
			print(await websocket.recv())

			await websocket.send('sendback "te\\"st"')
			time.sleep(0.1)
			print(await websocket.recv())

			await websocket.send('sendback "te\\\\"st"')
			time.sleep(0.1)
			print(await websocket.recv())
``` 
printed these results:
`te st`
`te"st`
`te\"st`

If you're certain that the extra logic is needed, by all means it can be put back. It just so happened in my case that I didn't encounter that situation and just opted to pass the payload 'as-is'